src/components/Docs/components/ForeignCoinsTable.tsx

```
@@ -34,13 +48,74 @@ const formatString = (str: string) => {
+    const wasAdded = await window.ethereum.request({
+      method: "wallet_watchAsset",
+      params: {
+        type: "ERC20",
+        options: {
+          address: token.zrc20_contract_address,
+        },
+      },
+    });
+
+    if (wasAdded) {
+      console.log("Token was added to MetaMask");
```

ESLint

[error] Unexpected console statement.

(no-console)